packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -330,8 +330,8 @@ class EditableText extends StatefulWidget {
        keyboardType = keyboardType ?? (maxLines == 1 ? TextInputType.text : TextInputType.multiline),
        inputFormatters = maxLines == 1
            ? (
-               <TextInputFormatter>[BlacklistingTextInputFormatter.singleLineFormatter]
-                 ..addAll(inputFormatters ?? const Iterable<TextInputFormatter>.empty())
+               <TextInputFormatter>[BlacklistingTextInputFormatter.singleLineFormatter, ...inputFormatters ?? const Iterable<TextInputFormatter>.empty()]
```

Comment:
Not quick fixable, but this would be better as `...?inputFormatters`

Reply:
We could have a lint and quick fix for that. We'd just need to be able to recognize some common patterns for "empty iterable".

Reply:
A quick grep of internal code shows quite a lot of `?? []` and `?? const []`.

Reply:
The assist should already handle that case.

Reply:
Oh man, we kept `...?` ?